jwst/ami/tests/test_ami_interface.py

```diff
@@ -7,28 +7,29 @@
 from jwst.ami import AmiAnalyzeStep
 
 
-def test_ami_analyze_cube_fail():
-    """Make sure ami_analyze fails if input is CubeModel (_calints)"""
+@pytest.fixture()
+def example_model():
     model = datamodels.CubeModel((25, 19, 19))
```

Comment:
It might be useful to flush this out with a small-enough but compute-enough input model that can be passed to the step and have it successfully run. At the moment it's only used for tests that produce errors (to check those errors).